ui/src/dms/MultiDMEditModal.tsx

```diff
@@ -1,21 +1,101 @@
+  const clubId = useParams<{ id: string }>().id!;
+  const club = useMultiDm(clubId);
+  const count = club?.team.length;
+  const pendingCount = club?.hive.length;
+  const hasPending = pendingCount && pendingCount > 0;
+  const [editing, setEditing] = useState(false);
+  const setEditingCb = useCallback(
+    (v) => {
+      setEditing(v);
+    },
+    [setEditing]
+  );
 
   return (
     <Dialog defaultOpen onOpenChange={(open) => !open && dismiss()}>
-      <DialogContent showClose containerClass="max-w-lg">
-        <div className="sm:w-96">
-          <header className="flex items-center ">
-            <h2 className="text-xl font-bold">Edit Chat Info</h2>
-          </header>
-        </div>
-        <MultiDMInfoForm setOpen={() => dismiss()} />
-      </DialogContent>
+      {editing ? (
+        <DialogContent showClose containerClass="w-max-lg">
+          <div className="sm:w-96">
+            <header className="flex items-center ">
+              <h2 className="text-xl font-bold">Edit Chat Info</h2>
+            </header>
+            <MultiDMInfoForm
+              setEditing={setEditingCb}
+              setOpen={() => dismiss()}
+            />
+          </div>
+        </DialogContent>
+      ) : (
+        <DialogContent showClose containerClass="max-w-lg">
+          <div className="sm:w-96">
+            <header className="flex items-center ">
+              <h2 className="text-xl font-bold">Chat Info</h2>
+            </header>
+          </div>
+          {club && count && (
+            <div className="flex flex-col items-center space-y-5">
+              <MultiDmAvatar {...club.meta} size="huge" />
+              <div className="text-center">
+                <div className="text-gray-600">
+                  <span>{`${count} ${pluralize('Member', count)}${
+                    hasPending ? ', ' : ''
+                  }`}</span>
+                  {hasPending ? (
+                    <span className="text-blue">{pendingCount} Pending</span>
+                  ) : null}
+                </div>
+                <h2 className="text-lg font-semibold">{club.meta.title}</h2>
+              </div>
+              <button className="button" onClick={() => setEditing(true)}>
+                Edit Chat Info
+              </button>
+              <h3 className="-mb-2 w-full text-left text-lg font-semibold text-gray-400">
+                Members
+              </h3>
+              <div className="flex w-full flex-col space-y-1">
+                {club.team.map((ship) => (
+                  <div
+                    className="flex items-center space-x-2 rounded-lg p-2 hover:bg-gray-50"
+                    key={ship}
+                  >
+                    <Avatar size="small" ship={ship} />
+                    <div className="flex grow flex-col">
+                      <ShipName
+                        name={ship}
+                        showAlias={true}
+                        className="font-bold text-gray-800"
```

Comment:
this looks semibold in the mocks to me, can't tell though because it's unclickable in figma